task.sql

```diff
@@ -1,33 +1,55 @@
--- Create database and tables
-
 CREATE DATABASE ShopDB;
 USE ShopDB;
 
 CREATE TABLE Countries (
-    ID INT,
-    Name VARCHAR(50),
-    PRIMARY KEY (ID)
+  ID INT,
```

Comment:
Consider adding `AUTO_INCREMENT` to the `ID` column to automatically generate unique IDs for new entries.

Reply:
Done